packages/atomic/src/components/facets-v1/facet-value-checkbox/facet-value-checkbox.pcss

```diff
@@ -0,0 +1,24 @@
+/**
+ * @prop --atomic-facet-checkbox-size: Size of the checkbox.
+ */
+.checkbox {
+  width: var(--atomic-facet-checkbox-size, 16px);
+  height: var(--atomic-facet-checkbox-size, 16px);
```

Comment:
If somebody changes text size, should checkboxes resize too? If so, we should probably use `em` here.

Reply:
Used rem for all those variables, great proposition 👍